packages/richtext-lexical/src/features/indent/client/index.tsx

```diff
@@ -81,10 +86,49 @@ const toolbarGroups: ToolbarGroup[] = [
   ]),
 ]
 
+const IndentPlugin: PluginComponent<IndentFeatureProps> = (props) => {
+  const [editor] = useLexicalComposerContext()
+  const { disabledNodes } = props.clientProps
+
+  useEffect(() => {
+    if (!editor) {
+      return
+    }
+    return mergeRegister(
+      editor.registerCommand(
+        INDENT_CONTENT_COMMAND,
+        () => {
+          return $handleIndentAndOutdent((block) => {
+            if (!disabledNodes || !disabledNodes.includes(block.getType())) {
+              const indent = block.getIndent()
+              block.setIndent(indent + 1)
+            }
+          })
+        },
+        COMMAND_PRIORITY_LOW,
+      ),
+      editor.registerCommand(
+        OUTDENT_CONTENT_COMMAND,
+        () => {
```

Comment:
Given that this doesn't check for `disabledNodes`, do we need this? Doesn't the `TabIndentationPlugin` already do this?

Reply:
Yes, you are right! I plan to override outdent later because there is a bug there, but it is not used at the moment.